nextflow_schema.json

```
@@ -25,7 +25,7 @@
                 },
                 "taxid": {
                     "type": "string",
-                    "description": "The taxonomy ID of the species identified in classification results."
+                    "description": "The taxonomy ID of the species identified in classification results. A list of taxids separated by spaces."
```

```suggestion
                    "description": "The taxonomy ID of the species identified in the classification results or a list of taxids separated by spaces."
```

Feel free to ignore this change if not relevant.